From 49569d774a2c161bc4dd39e5ec4e26e6511c0ba9 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Fri, 5 Sep 2025 23:36:54 +0800
Subject: [PATCH 84/90] ia64: add prototype declaration for `void
 pci_adjust_legacy_attr()'

Suppress a build error with `-Werror=missing-prototypes'.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 arch/ia64/include/asm/pci.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/ia64/include/asm/pci.h b/arch/ia64/include/asm/pci.h
index fa8f545c24c9..c95e17441549 100644
--- a/arch/ia64/include/asm/pci.h
+++ b/arch/ia64/include/asm/pci.h
@@ -34,6 +34,10 @@ struct pci_vector_struct {
 #define ARCH_GENERIC_PCI_MMAP_RESOURCE
 #define arch_can_pci_mmap_wc()	1
 
+extern void pci_adjust_legacy_attr(struct pci_bus *bus,
+				   enum pci_mmap_state mmap_type);
+
+
 #define HAVE_PCI_LEGACY
 extern int pci_mmap_legacy_page_range(struct pci_bus *bus,
 				      struct vm_area_struct *vma,
-- 
2.25.1

